ADAL/src/request/ADAuthenticationRequest+Broker.m

```diff
@@ -238,22 +309,34 @@ - (void)callBroker:(ADAuthenticationCallback)completionBlock
     NSString* adalVersion = [ADLogger getAdalVersion];
     CHECK_FOR_NIL(adalVersion);
     
-    NSDictionary* queryDictionary = @{
-                                      @"authority": _context.authority,
-                                      @"resource" : _resource,
-                                      @"client_id": _clientId,
-                                      @"redirect_uri": _redirectUri,
-                                      @"username_type": _identifier ? [_identifier typeAsString] : @"",
-                                      @"username": _identifier.userId ? _identifier.userId : @"",
-                                      @"force" : _promptBehavior == AD_FORCE_PROMPT ? @"YES" : @"NO",
-                                      @"correlation_id": _correlationId,
+    NSDictionary* queryDictionary =
+    @{
+      @"authority"      : _context.authority,
+      @"resource"       : _resource,
+      @"client_id"      : _clientId,
+      @"redirect_uri"   : _redirectUri,
+      @"username_type"  : _identifier ? [_identifier typeAsString] : @"",
+      @"username"       : _identifier.userId ? _identifier.userId : @"",
+      @"force"          : _promptBehavior == AD_FORCE_PROMPT ? @"YES" : @"NO",
+      @"correlation_id" : _correlationId,
 #if TARGET_OS_IPHONE // Broker Message Encryption
-                                      @"broker_key": base64UrlKey,
+      @"broker_key"     : base64UrlKey,
 #endif // TARGET_OS_IPHONE Broker Message Encryption
-                                      @"client_version": adalVersion,
-									  BROKER_MAX_PROTOCOL_VERSION : @"2",
-                                      @"extra_qp": _queryParams ? _queryParams : @"",
-                                      };
+      @"client_version" : adalVersion,
+      BROKER_MAX_PROTOCOL_VERSION : @"2",
+      @"extra_qp"       : _queryParams ? _queryParams : @"",
+      };
+    
+    NSDictionary* resumeDictionary =
+  @{
+    @"authority"        : _context.authority,
+    @"resource"         : _resource,
+    @"client_id"        : _clientId,
+    @"redirect_uri"     : _redirectUri,
+    @"correlation_id"   : _correlationId,
+    };
+    
+    [[NSUserDefaults standardUserDefaults] setObject:resumeDictionary forKey:kAdalResumeDictionaryKey];
```

Comment:
The two paths are nearly identical... except one never got updated to support version 2, and will prompt install, so it was pretty easy to collapse the two into a single callBroker: